pkgs/by-name/nv/nvs/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  writableTmpDirAsHomeHook,
+  lib,
+}:
+buildGoModule (finalAttrs: {
+  pname = "nvs";
+  version = "1.10.5";
+
+  src = fetchFromGitHub {
+    owner = "y3owk1n";
+    repo = "nvs";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-Qp5c2F383Z0MxTtDt3wmLgxiAwfIJWupVDCePrBxNQI=";
+  };
+
+  vendorHash = "sha256-l2FdnXA+vKVRekcIKt1R+MxppraTsmo0b/B7RNqnxjA=";
+
+  # Completions
+  nativeBuildInputs = [
+    installShellFiles
+    writableTmpDirAsHomeHook
+  ];
+  postInstall = ''
+    export HOME=$TMPDIR
```

Comment:
writableTmpDirAsHomeHook removes the need for this
```suggestion
```

Reply:
damn, of course, forgot, thanks!